6. Developer Guides/Code Styleguide/README.md

```diff
@@ -4,4 +4,4 @@ order: 4
 
 # Rocket.Chat Code Styleguide
 
-- [Less Code Styleguide](less)
\ No newline at end of file
+- [Less Code Styleguide](https://github.com/Prakharsvnit/Rocket.Chat.Docs/blob/patch-1/6.%20Developer%20Guides/Code%20Styleguide/less.md)
```

Comment:
No, please see this file for example: https://github.com/RocketChat/Rocket.Chat.Docs/blob/master/6.%20Developer%20Guides/REST%20API/README.md